core/src/CabanaConfig.cmakein

```diff
@@ -0,0 +1,6 @@
+include(CMakeFindDependencyMacro)
+find_dependency(KOKKOS REQUIRED)
```

Comment:
@dalg24 the crux is here, if want to make this work before #163 is merge, we would need to install our `FindKOKKOS.cmake`, which is just ugly. So wait?